### DISCUSSION TITLE
Integrating NUC Plotter, Synology NAS, and Rasberry Pi 4 - Recommendations?

### DISCUSSION THREAD

**Discussion by KnowTheFuture** [Chia-Network/chia-blockchain#6141] [Q&A]
I was not planning on finding myself on the crypto-currency bandwagon, but the heights of Bitcoin caught my attention. I was looking at ASIC miners, but the price hikes of miners, noise (like running a Dyson 24-7) in a small home with a family, and the spike in electricity bills (I'm based in Japan, and price per kWh is nearly double the average price of the USA) were all massive drawbacks, which is were Chia caught my eye.

**Main Computer**
- M1 Mac Mini - Chia is installed and synched.
**Plotter**
- Intel NUC i7, 32GB, 480TB boot SSD, 2TB Corsair P600 Gen 4 PCiE SSD - as covered by [JM here](https://chiadecentral.com/nuc-small-form-factor-plotting-build/) yes, the heatsink sticks out, so I used some spacers from a Rasberry Pi and sealed the gap.
- OS: Ubuntu Server
- Currently running plot creation individually while learning. Will install Plotman, however it may be better to wait for upcoming pool support since those plots will need to be replotted.
**Storage**
- Synology DS920+ NAS - currently 21TB (3 x 8TB)
- Silverstone 5-bay HDD Enclosure (future use)
**Farmer**
- Rasberry Pi 4
- OS: Ubuntu Server
- Planning to connect to the NAS, either via USB 3.0-to-3.0 or ethernet cable.
**Network**
- All devices are connected to a switching hub that is mounted under my desk

**Questions/Issues**
- The Synology NAS supports Linux. Would you recommend connecting via USB or ethernet?
- What would you recommend (currently looking at Chia Docker) to push completed plots from the plotter to the NAS? 
- What am I missing?

**Comment by tiny0little** [comment]
> The Synology NAS supports Linux. Would you recommend connecting via USB or ethernet?

USB


> What would you recommend (currently looking at Chia Docker) to push completed plots from the plotter to the NAS?

NAS is not recommended, all drives should be locally connected

> What am I missing?

plotter CPU too weak

**Reply by KnowTheFuture** [reply]
Thank you @tiny0little Looking into a bit more, David wrote about his own experience with the DS920 [here](https://david-waiting.medium.com/earn-chia-cryptocurrency-with-your-synology-nas-a-step-by-step-guide-fb3343ec6df2). The NAS has its own CPU and can run Ubuntu Server in a VM, so it appears the Pi is redundant for now. It will come in handy when I'm ready to build additional machines.
